src/lightning_app/components/training.py

```diff
@@ -126,8 +126,8 @@ def __init__(
         Example::
 
             from lightning import LightningApp
-            from lightning.app.components.training import LightningTrainingComponent
-            from lightning.app.utilities.packaging.cloud_compute import CloudCompute
+            from lightning_app.components.training import LightningTrainingComponent
+            from lightning_app.utilities.packaging.cloud_compute import CloudCompute
```

Comment:
yes it is whole purpose of this PR

Reply:
so let's remove the change from the docs then?

Reply:
This is not the story we want to tell in the docs. These statements here are not executed, I don't see a problem with having it like this in the docs. 

All of our docs must say 

```
pip install lightning

import lightning
```
No pytorch lightning, no lightning_app.


This was very clearly communicated by William months ago. The fact that there exist standalone packages is just a side-product. 

@lantiga @tchaton for confirmation

Reply:
And in the generated docs this will still be the case! 
The only issue is that this way we cannot run doctests at all. The current plan is to have them as standalone packages in the repo to have easier testing and developing and generate a unified package with unified docs (where it will say `lightning.pytorch` and `lightning.app` out of these standalone ones. But in order to not have different conditions all over the place, we first need to unify the standalone structure as a requirement for the later package-generation!